kikar_hamedina/core/templatetags/core_extras.py

```diff
@@ -57,3 +60,21 @@ def path_to_params(path):
 @register.filter(name='language_clean_uri')
 def language_clean_uri(uri):
     return uri.split('/en')[-1].split('/ar')[-1].split('/he')[-1]
+
+@register.filter(name='naturaltime_ext')
+def naturaltime_ext(value, locale):
+    formatted_value = ''
+
+    if locale != 'he':
+        formatted_value = 'Published: '
+
+    time_differ = timezone.now() - value
+    if time_differ.days < 7:
+        formatted_value += naturaltime(value)
+    else:
+        if locale != 'he':
+            formatted_value += format(value, 'F jS, Y, g a')
```

Comment:
Hi,

I'm merging it in for the moment.
But try to rewrite this part using translation strings.
see example here: https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#standard-translation , there's an example with handling dates :)